README.md

```diff
@@ -55,32 +61,57 @@ After you have installed GOV.UK Frontend the `@govuk-frontend` package will appe
 
 ## Import assets
 
-Import styles into your main Sass file.
+### Import styles
+
+You need to import the GOV.UK Frontend styles into the main Sass file in your project.
 
-To import all components, add the line:
+1. To import all components, add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/all/all";
 ```
 
-To import an individual component (for example, a button), add the line:
+2. To import an individual component (for example a button), add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/button/button";
 ```
 
+You should place the above code before your other Sass rules, or Sass imports, if you want to override
+GOV.UK Frontend with your own styles.
 
-To resolve your `@import` declarations you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) or [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths) in Ruby.
+In order to resolve the above `@import` paths(s) in your build, you should add `node_modules` to your [Sass include paths](https://github.com/sass/node-sass#includepaths) (in Ruby, they should be added to [assets paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths)).
+
+For example, if your project uses Gulp, you would add the Sass include paths to your Gulp configuration file (for example `gulpfile.js`) with [gulp-sass](https://www.npmjs.com/package/gulp-sass). Below is an example:
 
-Below is a code sample you can add to your gulp configuration file using the gulp-sass package:
 ```JS
-.pipe(sass({ includePaths: 'node_modules/' }))
+gulp.task('sass', function () {
+  return gulp.src('./sass/**/*.scss')
+    .pipe(sass({
+      includePaths: 'node_modules/@govuk-frontend'
+     }))
+    .pipe(gulp.dest('./css'));
+});
+
 ```
 
-To import images, configure your application to reference or copy the icons assets.
+Note: If you compile Sass to CSS in your project, your build tasks will already include something similar to the above task - in
+that case, you will just need to include add `includePaths` to it.
+
+### Import images and icons
+
+In order to import GOV.UK Frontend images and icons to your project, you should configure your application to reference or copy the relevant GOV.UK Frontend assets.
 
-Below is a code sample you can add to your [express.js](https://expressjs.com/) configuration:
+Recommended solution:
+
+Make `/node_modules/@govuk-frontend/icons` available to your project by routing requests for your images folder there. For example, if your project uses [express.js](https://expressjs.com/), below is a code sample you could add to your configuration:
```

Comment:
Could we have the example on a new line, as we did on the _Import styles_ section?

Reply:
Done!